life_expectancy/tests/test_load_save_data.py

```diff
@@ -1,15 +1,23 @@
 """Tests for the load and save module"""
 from unittest.mock import patch
 import pandas as pd
-from life_expectancy.load_save_data import load_data, save_data
+from life_expectancy.load_save_data import LoadJsonStrategy, LoadCsvStrategy, save_data
 from . import OUTPUT_DIR
 
 
 @patch("life_expectancy.load_save_data.pd.read_table")
-def test_load_data(mock_read_csv, eu_life_expectancy_raw) -> None:
+def test_load_json(mock_read_json, eurostat_life_expect) -> None:
+    """test load_data is a panda dataframe"""
+    mock_read_json.return_value = eurostat_life_expect
+    strategy = LoadJsonStrategy()
+    result = strategy.load_data()
+    assert isinstance(result, pd.DataFrame)
+
+@patch("life_expectancy.load_save_data.pd.read_csv")
+def test_load_csv(mock_read_csv, eu_life_expectancy_raw) -> None:
     """test load_data is a panda dataframe"""
     mock_read_csv.return_value = eu_life_expectancy_raw
-    result = load_data()
+    result = LoadCsvStrategy().load_data()
```

Comment:
I think it is possible to join this 2 load tests in one using @pytest.mark.parametrize